extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js

```diff
@@ -385,7 +399,8 @@ export class AmpAdNetworkDoubleclickImpl extends AmpA4A {
   getBlockParameters_() {
     dev().assert(this.initialSize_);
     dev().assert(this.jsonTargeting_);
-    let sizeStr = `${this.initialSize_.width}x${this.initialSize_.height}`;
+    let sizeStr = this.isFluid_ ?
+        '320x50' : `${this.initialSize_.width}x${this.initialSize_.height}`;
```

Comment:
If we're fluid we fix the size at 320x50?  This is required?

Reply:
Yes, it's required.